src/txdb.cpp

```diff
@@ -28,6 +28,10 @@ static const char DB_FLAG = 'F';
 static const char DB_REINDEX_FLAG = 'R';
 static const char DB_LAST_BLOCK = 'l';
 
+//! Used to cache the nChainTx value for the base block of a UTXO snapshot.
```

Comment:
In commit "txdb: add nChainTx cache for snapshot chainstates" (bc622c33ce925159906e3b00be77b11d598397ed)

Can you remind what this is used for? Just estimating sync progress, or other things too? Would also be good to make comment more specific, maybe "Cached CBlockIndex::nChainTx value (number of transactions in chain) at the base block of a UTXO snapshot that is being imported. This is used to [...]"